src/pilot/cpu_siblings.py

```diff
@@ -16,45 +16,221 @@
 
 
 # It is a dictionary of the form {'total number of cpu cores' :
-# {'number_of_host_os_cpu' : {'vcpu_pin_set' : 'Range of corresponding cpus',
-# 'host_os_cpu' : 'Range of corresponding cpus'} } }
+# {'number_of_host_os_cpu' : {
+# 'vcpu_pin_set' : 'Range of corresponding cpus',
+# 'host_os_cpu' : 'Range of corresponding cpus',
```

Comment:
I would prefer "Range of corresponding cpus available for Host OS"

Reply:
Fixed